cypress/e2e/components/headernavigation.cy.js

```diff
@@ -1,27 +1,24 @@
-import selectors from '../constants/selectors';
+import selectors, { componentSelectors } from '../constants/selectors';
 import { routes } from '../constants/url';
 
-Cypress.Commands.add('inHeader', (callback) => {
-    cy.get('header').within(callback);
-});
-
-describe('navigation area and navigation items', () => {
+describe('check navigation area, navigation items and link goals', () => {
     beforeEach(() => {
         cy.visit(routes.home);
     });
     context('navigation area and linkgoals', () => {
-        it('check navlinks', () => {
-            cy.inHeader(() => {
-                cy.get('nav').should('be.visible');
-            });
+        it('check navigation area and navigation items', () => {
+            cy.get(componentSelectors.navigation)
+                .should('be.visible')
+                .within(() => {
+                    cy.get('ul')
```

Comment:
```suggestion
                    cy.get(componentSelectors.navigationItem)
```
wir haben ein data-component für die links. Die benutzt du auch unten

Reply:
data component verwendet